tests/drivers/spi/spi_controller_peripheral/boards/frdm_mcxa156.overlay

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <lpspi1_header.overlay>
+
+/ {
+	zephyr,user {
+		peripheral-cs = <1>;
+	};
+};
+
```

Comment:
It would be useful for testing if you could include the connection setup in a comment here.